addons/udes_delivery_control/views/stock_picking.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_form_stock_picking" model="ir.ui.view">
+        <field name="name">udes_delivery_control_form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
```

Comment:
Can we hide the Reserved Pallet field from `udes_stock` for Delivery Control pickings? Can't imagine it is relevant. :)

![image](https://user-images.githubusercontent.com/7999677/96563713-2fcb4600-12ba-11eb-895c-b58a4cbce1fd.png)